frontend/.eslintrc.json

```diff
@@ -1,10 +1,5 @@
 {
     "extends": "react-app",
     "rules": {
```

Comment:
I think you could probably remove this file entirely, since it's no longer overriding anything anymore.

Reply:
I guess maybe we could not remove the configuration file, since it might cause the error: 
![Screen Shot 2020-10-01 at 4 08 30 PM](https://user-images.githubusercontent.com/34461466/94872142-af26d180-0400-11eb-939c-713549261770.png)

Reply:
Oh, ok, got it